tests/test_log.py

```diff
@@ -1,6 +1,39 @@
+import logging
+
 import pytest
 
-from mandible.log import log_errors
+from mandible.log import init_custom_log_record_factory, log_errors
+
+
+@pytest.fixture
+def event_config():
+    return {
+        "cma": {
+            "event": {
+                "cumulus_meta": {
+                    "execution_name": "this_is_a_test_execution",
+                    "cumulus_version": "TEST_CUMULUS"
+                },
+                "payload": {
+                    "granules": [{"granuleId": "test_granule"}]
+                }
+            }
+        }
+    }
+
+
+def test_custom_log_record_factory(caplog, monkeypatch, event_config):
+    monkeypatch.setenv("DAAC_VERSION", "TEST_DAAC")
+    monkeypatch.setenv("CORE_VERSION", "TEST_CORE")
+    caplog.set_level(logging.INFO)
```

Comment:
I like using `caplog.at_level` instead to only change the log level for the duration that you want to be testing so that any other helpful debug messages will still show up in the pytest output if something fails.